auth/src/main/java/com/redhat/lightblue/rest/auth/CachedRolesProvider.java

```diff
@@ -72,4 +74,9 @@ public CachedRolesProvider(RolesProvider rolesProvider, RolesCache rolesCache) {
         return roles;
     }
 
+    @Override
+    public RolesProviderHealth checkHealth() {
+        return rolesProvider.checkHealth();
+    }
```

Comment:
Maybe eventually we need health check to not be so binary but instead more like "healthy, degraded, or unhealthy." Seems complex. Hrm!